install_files/ansible-base/roles/app/templates/sites-available/source.conf

```diff
@@ -1,5 +1,23 @@
-ServerName {{ apache_listening_address | default('127.0.0.1') }}
-<VirtualHost {{ apache_listening_address | default('127.0.0.1') }}:80>
+ServerName {{ securedrop_app_apache_listening_address }}
+<VirtualHost {{ securedrop_app_apache_listening_address }}:80>
+{% if securedrop_app_https_on_source_interface %}
+# Redirect to HTTPS, and comment out the rest of this section
+RewriteEngine On
+RewriteCond %{HTTPS} off
+RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI}
+</VirtualHost>
+
+<VirtualHost 127.0.0.1:443>
+SSLEngine on
+SSLCertificateFile /var/lib/ssl/{{ securedrop_app_https_certificate_cert_src|basename }}
+SSLCertificateKeyFile /var/lib/ssl/{{ securedrop_app_https_certificate_key_src|basename }}
+SSLCertificateChainFile /var/lib/ssl/{{ securedrop_app_https_certificate_chain_src|basename }}
+
+SSLProtocol all -SSLv2 -SSLv3
+SSLHonorCipherOrder on
+SSLCompression off
+SSLCipherSuite EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:!MD5
```

Comment:
Will make a followup issue for updating the ciphersuites, but in the interest of incremental progress I'm not going to block merge on this